pep-XXXX.rst

```diff
@@ -0,0 +1,258 @@
+-----------------------------------
+
+Even if author or maintainers of a Python extension module with to use no
+external shared libraries, the modules will generally have a dynamic runtime
+dependency on the GNU C library, ``glibc``. While it is possible, statically
+linking ``glibc`` is usually a bad idea because of bloat, and because certain
+important C functions like ``dlopen()`` cannot be called from code that
+statically links ``glibc``. A runtime shared library dependency on a
+system-provided ``glibc`` is unavoidable in practice.
+
+The maintainers of the GNU C library follow a strict symbol versioning scheme
+for backward compatibility. This ensures that binaries compiled against an older
+version of ``glibc`` can run on systems that have a newer ``glibc``. The
+opposite is generally not true -- binaries compiled on newer Linux
+distributions tend to rely upon versioned functions in glibc that are not
+available on older systems.
+
+This generally prevents built distributions compiled on the latest Linux
+distributions from being portable.
+
+
+The ``manylinux_1`` policy
+==========================
+
+For these reasons, to achieve broad portability, Python wheels
+
+ * should depend only on an extremely limited set of external shared
+   libraries; and
+ * should depend only on ``old`` symbol versions in those external shared
+   libraries.
+
+The ``manylinux_1`` policy thus encompasses a standard for what the
+permitted external shared libraries a wheel may depend on, and the maximum
+depended-upon symbol versions therein.
+
+The permitted external shared libraries are: ::
+
+  "libpanelw.so.5", "libncursesw.so.5", "libgcc_s.so.1", "libstdc++.so.6",
+  "libm.so.6", "libdl.so.2", "librt.so.1", "libcrypt.so.1", "libc.so.6",
+  "libnsl.so.1", "libutil.so.1", "libpthread.so.0", "libX11.so.6",
+  "libXext.so.6", "libXrender.so.1", "libICE.so.6", "libSM.so.6",
+  "libGL.so.1", "libgobject-2.0.so.0", "libgthread-2.0.so.0",
+  "libglib-2.0.so.0", "libgmodule-2.0.so.0", and "libgio-2.0.so.0".
+
+On Debian-based systems, these libraries are provided by the packages ::
+
+    "libncurses5", "libgcc1", "libstdc++6", "libc6", "libx11-6", "libxext6",
+    "libxrender1", "libice6", "libsm6", "libgl1-mesa-glx", and "libglib2.0-0".
+
+On RPM-based systems, these libraries are provided by the packages ::
+
+    "ncurses", "libgcc", "libstdc++", "glibc", "libXext", "libXrender",
+    "libICE", "libSM", "mesa-libGL", and "glib2".
+
+This list was compiled by checking the external shared library dependencies of
+the Canopy [1]_ and Anaconda [2]_ distributions, which both include a wide array
+of the most popular Python modules and have been confirmed in practice to work
+across a wide swath of Linux systems in the wild.
+
+For dependencies on externally-provided versioned symbols in the above shared
+libraries, the following symbol versions are permitted: ::
+
+    GLIBC <= 2.5
+    CXXABI <= 3.4.8
+    GLIBCXX <= 3.4.9
+    GCC <= 4.2.0
```

Comment:
(That gist is for libncursesw since that's the version in the current profile, but I see similar results for libncurses-without-the-w, so that's not the issue)